src/View/Helper/FormHelper.php

```diff
@@ -31,7 +31,6 @@ class FormHelper extends Helper
         'help' => '<div class="help-block">{{content}}</div>',
         'inputContainer' => '<div class="form-group {{type}}{{required}}">{{content}}{{help}}</div>',
         'inputContainerError' => '<div class="form-group {{type}}{{required}} has-error">{{content}}{{error}}{{help}}</div>',
-        'checkboxWrapper' => '<div class="checkbox"><label>{{input}}{{label}}</label></div>',
         'multipleCheckboxWrapper' => '<div class="checkbox">{{label}}</div>',
```

Comment:
That should better become a separate cleanup issue/task.
This fix here is isolated and well tested, whereas the multipleCheckboxWrapper thing seems to be somewhat entangled in the current code still:

```php
case 'select':
                if (isset($options['multiple']) && $options['multiple'] === 'checkbox') {
                    $options['templates']['checkboxWrapper'] = $this->templater()->get('multipleCheckboxWrapper');
                    $options['type'] = 'multicheckbox';
                }
                break;
```